packages/store/addon/-private/ts-interfaces/minimum-adapter-interface.ts

```diff
@@ -0,0 +1,97 @@
+/**
+  ## Overview
+
+  In order to properly fetch and update data, EmberData
+  needs to understand how to connect to your API.
+
+  `Adapters` accept various kinds of requests from the store
+  and manage fulfillment of the request from your API.
+
+  ### Implementing an Adapter
+
+
```

Comment:
Do we just steal something from the `/adapter` package?

Reply:
possibly some language there we can steal, would like to give it a new write up though similar to the serializer package docs